.github/workflows/tests.yml

```diff
@@ -66,7 +44,7 @@ jobs:
     - name: Install dependencies
       run: |
         python -m pip install --upgrade pip
-        python -m pip install --upgrade tox tox-gh-actions testfixtures
+        python -m pip install --upgrade 'tox>=3,<4' tox-gh-actions testfixtures
```

Comment:
Version 4 of tox came out a few weeks ago and the CI doesn't work with it. I've pinned to tox 3 for now.

Reply:
Sounds good, once tox 4 is working we can explicitly reference that version.